ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,24 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New Fields for PAC (ACU) Units
+- return_air_isolation_damper_status
+	- OPEN
+	- CLOSE
+- alert_reset_command
+	- NORMAL
+	- RESET
+- fan_run_time_accumulator
```

Comment:
Which fan?

Reply:
The PAC unit consists of  compressors and a Fan. Once the set point temperature is achieved the compressor is shut off and the Fan switches on.

Reply:
so a discharge fan to a space? if so, `discharge_fan_run_time_accumulator`